HoloStorageAccessor/api_spec/openapi.yaml

```diff
@@ -63,24 +66,31 @@ paths:
         content:
           application/json:
             schema:
-              $ref: '#/components/schemas/PatientBasic'
+              $ref: '#/components/schemas/Patient'
       responses:
         200:
           description: Successful modification
         201:
           description: Successful created
-        400:
-          description: Unaccepted input
+        500:
+          $ref: "#/components/responses/InternalError"
 
   /holograms:
     get:
       summary: Mass query for hologram metadata based on hologram ids
       description: |-
         Obtain hologram information stored within HoloStorage.
 
-        Multiple IDs can be passed with the syntax `/holograms?hid=id1,id2,id3,...,idx`
+        Multiple IDs can be passed with the syntax `/holograms?hid=id1,id2,id3,...,idx` or `/holograms?pid=id1,id2,id3,...,idx`
+
+        Additional filtering for CreationMode can be done with `&creationmode=...`
+
+        Using neither `?pid` nor `?hid` will result in an empty result set.
```

Comment:
Could mention that using both at once is also not allowed

Reply:
This has been added in a note above.